### PR TITLE
Updated technical writing handbook 

### PR DESCRIPTION
**Pull request by justinegeffen**
Updated AORs, updated review areas, added a tips and best practices section thanks to @mickmister. :)

### CODE REVIEW

File: operations/research-and-development/product/technical-writing-team-handbook/README.md [hunk 1]
```diff
@@ -213,41 +206,50 @@ For best results, please include as much content, context, and detail as possibl
...
-#### Engineering/developer documentation
+### Engineering/developer documentation
 
 Most, if not all, contributions to the Mattermost project have a documentation impact. As part of the development and submission process, it’s recommended that the relevant documentation be updated \(or created\) and included in the PR. This provides consistency and accuracy in communicating the changes/new feature and cuts down on having multiple issues and PRs for related documentation.
 
 Visit the [Contribute to Documentation](https://handbook.mattermost.com/operations/operations/publishing/publishing-guidelines/voice-tone-and-writing-style-guidelines/submitting-documentation-with-your-pr) page to get started with submitting documentation with your PR. You can read more about the review process [here](https://developers.mattermost.com/contribute/getting-started/code-review).
 
+## PR tips and best practices
+
+* Try submit files in batches rather than single PRs per file. This reduces the PR load and also groups your changes more effectively.
```
Comment (mickmister) [inline operations/research-and-development/product/technical-writing-team-handbook/README.md]:
```suggestion
* Try to submit files in batches rather than single PRs per file. This reduces the PR load and also groups your changes more effectively.
```

File: operations/research-and-development/product/technical-writing-team-handbook/README.md [hunk 2]
```diff
@@ -213,41 +206,50 @@ For best results, please include as much content, context, and detail as possibl
...
+* When there are lots of changes requested, especially on a docs PR where the changes are "safe" (meaning they don't need to be QA tested), you can use the "Add suggestion to batch" feature in GitHub to commit all of the suggestions at once. You have to be in the "Files changed" tab to actually use the feature, though it is also shown in the "Conversation" tab. Then you can use one commit message like "Added batch suggestions from @Carrie Warner". This just makes the commit history of the PR a lot more readable and useful.
+* Descriptive PR titles and commit messages go a long way, especially when the PR is referenced in the future. The default PR titles that GitHub gives us like "Updated README.md" are not very helpful because there's no way to know the purpose of the PR by reading the title. For all the reviewer knows, maybe the whole file was translated to Japanese because we realize that's our main audience, but I wouldn't know if that was the case unless it was in the PR title.
```
Comment (mickmister) [inline operations/research-and-development/product/technical-writing-team-handbook/README.md]: > For all the reviewer knows, maybe the whole file was translated to Japanese because we realize that's our main audience, but I wouldn't know if that was the case unless it was in the PR title.

This sort of language was meant to be a joke 🙂  Especially since this says "I", this wording doesn't seem appropriate for documentation. I'm not sure if we want to reword this to convey the same thing or just remove the sentence. @cwarnermm What are your thoughts?

File: operations/research-and-development/product/technical-writing-team-handbook/README.md [hunk 3]
```diff
@@ -213,41 +206,50 @@ For best results, please include as much content, context, and detail as possibl
...
+
+## GitHub training
+
+GitHub training videos coming soon!
```
Comment (mickmister) [inline operations/research-and-development/product/technical-writing-team-handbook/README.md:255]: Should we have a ticket for this? Maybe we should have a ticket for now instead of this point here, because we don't know where they will exist or if they will be made in the format of videos, for instance.

Also, I think the point about descriptive PR titles and commit messages would also be very helpful to developer contributions, as well as the upcoming GitHub training information. I think this information should be readily available to all kinds of contributors, and so it may be missed if it's on this page. Also, I don't want these tips to be missed in general because they're buried on the bottom of this page.

File: operations/research-and-development/product/technical-writing-team-handbook/README.md [hunk 4]
```diff
@@ -8,24 +8,21 @@ The Mattermost Technical Writers focus primarily on writing, editing, and mainta
...
   * Self-managed deployments
   * Messaging features and functionality
-* Incident Collaboration
-* Focalboard
+* Playbooks
+* Boards
 * Growth
   * Customer Portal
   * Telemetry
+* Apps Framework and Marketplace
```
Comment (mickmister) [inline operations/research-and-development/product/technical-writing-team-handbook/README.md]: Although we often call it the Apps Framework internally, I think we landed on App Framework for external advertising.

```suggestion
* App Framework and Marketplace
```

**Review by mickmister — CHANGES_REQUESTED** [review]

**Comment by cwarnermm** [comment]
It may be time to update this line: "The Mattermost Technical Writers focus primarily on writing, editing, and maintaining technical documentation \(including the User's Guide, Administrator's Guide, and Developer documentation\)", given that we no longer have those guides in production.

File: operations/research-and-development/product/technical-writing-team-handbook/README.md (hunk 2 above)
Comment (cwarnermm) [reply operations/research-and-development/product/technical-writing-team-handbook/README.md]: I agree, @mickmister, that using first person voice is at odds with our Mattermost voice overall. However, I do think we should keep this messaging in and tweak the voice for consistency because it's a great point and worthwhile context to share with readers.

File: operations/research-and-development/product/technical-writing-team-handbook/README.md (hunk 3 above)
Comment (cwarnermm) [reply operations/research-and-development/product/technical-writing-team-handbook/README.md:255]: Another route we could take right away is finding some good quality, reputable GitHub training material online and providing links to that, rather than creating our own. GitHub itself may have such material we could point to. @aspleenic and the Community Team may also have additional resources they could recommend.

**Review by cwarnermm — DISMISSED** [review]
This is going to be a GREAT update to the Handbook! Provided inline thoughts, comments, and suggestions.

File: operations/research-and-development/product/technical-writing-team-handbook/README.md (hunk 2 above)
Comment (justinegeffen) [reply operations/research-and-development/product/technical-writing-team-handbook/README.md]: This is my bad - I didn't thoroughly read it before pushing the button.

File: operations/research-and-development/product/technical-writing-team-handbook/README.md [hunk 5]
```diff
@@ -213,41 +206,50 @@ For best results, please include as much content, context, and detail as possibl
...
 
+## PR tips and best practices
+
+* Try to submit files in batches rather than single PRs per file. This reduces the PR load and also groups your changes more effectively.
+* When there are lots of changes requested, especially on a docs PR where the changes are "safe" (meaning they don't need to be QA tested), you can use the "Add suggestion to batch" feature in GitHub to commit all of the suggestions at once. You have to be in the "Files changed" tab to actually use the feature, though it is also shown in the "Conversation" tab. Then you can use one commit message like "Added batch suggestions from @Carrie Warner". This just makes the commit history of the PR a lot more readable and useful.
+* Descriptive PR titles and commit messages go a long way, especially when the PR is referenced in the future. The default PR titles that GitHub gives us like "Updated README.md" are not very helpful because there's no way to know the purpose of the PR by reading the title. For all the reviewer knows, maybe the whole file was translated to Japanese because we realize that's our main audience, but I wouldn't know if that was the case unless it was in the PR title.
```
Comment (justinegeffen) [inline operations/research-and-development/product/technical-writing-team-handbook/README.md]:
```suggestion
* Descriptive PR titles and commit messages go a long way, especially when the PR is referenced in the future. The default PR titles that GitHub gives us like "Updated README.md" are not very helpful because there's no way to know the purpose of the PR by reading the title. For example, if the whole file was translated to Japanese because we realize that's our main audience, that information isn't obvious unless I open the PR and read it - or if it's in the PR title.
```

File: operations/research-and-development/product/technical-writing-team-handbook/README.md (hunk 3 above)
Comment (justinegeffen) [reply operations/research-and-development/product/technical-writing-team-handbook/README.md:255]: That's a great question, @mickmister! And I noticed that Carlos +1ed the idea about GH training videos. It is something I planned to raise in the Handbook Champions sync - but as the Champions pool is currently small, I can raise an issue in an appropriate repo and crowdsource some help.

Comment (mickmister) [reply operations/research-and-development/product/technical-writing-team-handbook/README.md:255]: I agree with @cwarnermm that a quick action would be to link to some GitHub documentation such as https://docs.github.com/en/github/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/creating-a-pull-request.

Creating videos on the topic of contributing, catered to our projects specifically, sounds very helpful to newcomers. As far as mentioning the videos here right now, 0/5 I think posting links to GitHub docs is good for now.

What do you think about the second part of my comment above pertaining to surfacing these tips to be more visible than the bottom of this page, with some of the tips also placed on developer docs?

**Review by mickmister — DISMISSED** [review]
LGTM :+1:

File: operations/research-and-development/product/technical-writing-team-handbook/README.md (hunk 3 above)
Comment (justinegeffen) [reply operations/research-and-development/product/technical-writing-team-handbook/README.md:255]: I'm all for moving the tips higher up for visibility! :)

File: operations/research-and-development/product/technical-writing-team-handbook/README.md [hunk 6]
```diff
@@ -2,7 +2,7 @@
 
 ## Technical Writing Handbook
 
-The Mattermost Technical Writers focus primarily on writing, editing, and maintaining technical documentation \(including the User's Guide, Administrator's Guide, and Developer documentation\).
+The Mattermost Technical Writers focus primarily on writing, editing, structuring, and maintaining technical and product documentation for Mattermost Channels, Boards, and Playbooks as well as developer documentation and the company Handbook. Technical Writers also work on in-product microcopy, provide editorial reviews for all documentation across the organization, and work closely with the community to improve documentation.
```
Comment (cwarnermm) [inline operations/research-and-development/product/technical-writing-team-handbook/README.md]:
```suggestion
Mattermost Technical Writers focus primarily on writing, editing, structuring, and maintaining Mattermost technical and product documentation, end user documentation for Mattermost Channels, Boards, and Playbooks, as well as developer documentation and the company Handbook. Technical Writers also manage in-product microcopy, provide editorial reviews for all documentation across the organization, and work closely with the community to improve documentation.
```

File: operations/research-and-development/product/technical-writing-team-handbook/README.md [hunk 7]
```diff
@@ -52,6 +52,12 @@ The basic outline for getting started with contributions is provided in the [REA
 
 Once complete, submit your Pull Request \(PR\). Ensure that you assign appropriate approvers and labels - if you've forked the repo or don't have the correct repo permissions you may not be able to do this, so you can skip this step. Read more about the review process [here](https://developers.mattermost.com/contribute/getting-started/code-review) - it applies to various types of contributions including documentation.
 
+## PR tips and best practices
+
+* Try to submit files in batches rather than single PRs per file. This reduces the PR load and also groups your changes more effectively.
```
Comment (cwarnermm) [inline operations/research-and-development/product/technical-writing-team-handbook/README.md]:
```suggestion
* Try to submit files in batches rather than single PRs for each file update. This reduces the PR load and also groups your changes more effectively.
```

File: operations/research-and-development/product/technical-writing-team-handbook/README.md [hunk 8]
```diff
@@ -52,6 +52,12 @@ The basic outline for getting started with contributions is provided in the [REA
...
+* When there are lots of changes requested, especially on a docs PR where the changes are "safe" (meaning they don't need to be QA tested), you can use the "Add suggestion to batch" feature in GitHub to commit all of the suggestions at once. You have to be in the "Files changed" tab to actually use the feature, though it is also shown in the "Conversation" tab. Then you can use one commit message like "Added batch suggestions from @Carrie Warner". This just makes the commit history of the PR a lot more readable and useful.
```
Comment (cwarnermm) [inline operations/research-and-development/product/technical-writing-team-handbook/README.md]:
```suggestion
* When there are lots of changes requested, especially on a docs PR that doesn't require QA testing, you can use the "Add suggestion to batch" feature in GitHub to commit all of the suggestions at once. You need to be on the **Files changed** tab to use this feature, though it's also shown on the **Conversation** tab. By batching your suggestions, you can use one commit message such as "Added batch suggestions from @Carrie Warner". This just makes the commit history of the PR a lot more readable and useful for everyone reviewing it.
```

**Review by cwarnermm — DISMISSED** [review]
Great updates! I've provided a couple of suggestions to consider. Approving for expediency.

**Review by cwarnermm — APPROVED** [review]
